packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -504,6 +504,26 @@ QUnit.module('interaction via swipe', {
         });
     });
 
+    QUnit.test('when all items are hidden selectedIndex should be initialized to 0 after swipe', function(assert) {
```

Comment:
You are fixing not this, you're fixing infinite loop error. Moreover, it seems like setting selectedIndex to 0 when all items are hidden are absolutely not related to the code you re changing, it's made separately [there](https://github.com/DevExpress/DevExtreme/pull/28299/files#diff-b95a070f9f0a64ed59233db53b817dce631e45fc0a7e5c12ff624c54ffd8b93aR164)
```suggestion
    QUnit.test('there should not be infinite loop if try swiping when all items are hidden ', function(assert) {
```

Reply:
[563cea3](https://github.com/DevExpress/DevExtreme/pull/28299/commits/563cea32b40366930aa50fa71d7ae64a401c7b3a)